### PR TITLE
Mkerns/caseflow 1611 unrecognized appellant btn hide

### PR DESCRIPTION
**Pull request by mkerns355**
Resolves [CASEFLOW-1611](https://vajira.max.gov/browse/CASEFLOW-1611)

### Description
Adds conditional logic on to the Power Of Attorney Details section to show/hide the refresh button based on if there is an Unrecognized Appellant or Power Of Attorney.

## Test Scenarios
- [ ] Recognized appellant w/ recognized POA - Show Button
- [ ] Unrecognized appellant w/ recognized POA - Do NOT show button
- [ ] Unrecognized appellant w/unrecognized POA - Do NOT show button

### Acceptance Criteria
- [ ] Code compiles correctly
- [ ] Tests pass

### CODE REVIEW

File: client/app/queue/PowerOfAttorneyDetail.jsx [hunk 1]
```diff
@@ -123,10 +124,11 @@ export const PowerOfAttorneyDetailUnconnected = ({ powerOfAttorney, appealId, po
   return (
     <React.Fragment>
       <div>
-        <p>{ poa.representative_type === 'Unrecognized representative' ?
-          <em>{ COPY.CASE_DETAILS_UNRECOGNIZED_POA }</em> :
-          <PoaRefresh powerOfAttorney={poa} appealId={appealId} {...detailListStyling} />}
-        </p>
+        { appellantType !== 'OtherClaimant' && (
+          poa.representative_type === 'Unrecognized representative' ?
```
Comment (ajspotts) [inline client/app/queue/PowerOfAttorneyDetail.jsx:128]: So will the button show if it is an unrecognized appellant with a recognized POA? My understanding is that regardless of POA if it is an unrecognized appellant we don't want to show the button

Comment (mkerns355) [reply client/app/queue/PowerOfAttorneyDetail.jsx:128]: Nope that first check is catching that scenario. So if the Appellant is unrecognized its not stepping inside to the POA check it just ignores it and doesn't show the button.

Comment (ajspotts) [reply client/app/queue/PowerOfAttorneyDetail.jsx:128]: Ah ok, so out of the 4 scenarios:
- Recognized appellant w/ recognized POA
- Recognized appellant w/ unrecognized POA
- Unrecognized appellant w/ recognized POA
- Unrecognized appellant w/unrecognized POA

We are only showing the button under the first scenario is that correct?

Comment (mkerns355) [reply client/app/queue/PowerOfAttorneyDetail.jsx:128]: Correct!

**Review by Jeremy1026 — APPROVED** [review]

**Pull request merged**